### PR TITLE
fix:[CORE-000] add logger statements to qualys-kb collector

### PR DESCRIPTION
**Pull request by raviteja-mandala**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List
any dependencies that are required for this change.

Fixes # (issue)

Added logger statements in qualys to debug the data alert issue.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore (no code changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

### CODE REVIEW

File: jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java [hunk 1]
```diff
@@ -70,9 +70,15 @@ public static Map<String, Object> execute(Map<String, String> params) throws Nam
             break;
         }
         ArrayList errors= (ArrayList) errorInfo.get("errors");
+
         if(!errors.isEmpty()){
             //Below logger message is used by datadog to create notification in slack
             log.error("Error occurred in atleast one collector for jobId : {}",jobHint);
```
Comment (plyubich) [inline jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java:76]: Spelling: atleast -> at least
space between ,jobHint

File: jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/util/ErrorManageUtil.java [hunk 2]
```diff
@@ -31,10 +35,13 @@ public static Map<String,Object> formErrorCode(List<Map<String,String>> errorLis
             for(Map<String, String> errorDetail :errorList) {
                 Map<String,Object> error = new HashMap<>();
                 error.put(ERROR, errorDetail.get(ERROR));
+                log.error("error detail - {}",errorDetail.get(ERROR));
```
Comment (plyubich) [inline jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/util/ErrorManageUtil.java:38]: Please be consistent with logging messages.

Update error msg to
"Error details : {}"

space between ,errorDetail.

File: jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java [hunk 3]
```diff
@@ -70,9 +70,15 @@ public static Map<String, Object> execute(Map<String, String> params) throws Nam
...
+            errors.stream().forEach(error -> {
+                Map<String,Object> err = (Map<String,Object>)error;
+                log.error("error in Main - {}",err.get("error"));
```
Comment (plyubich) [inline jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java:79]: Please be consistent with letter case consistency

error in Main -> Error in Main : {}

File: jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java [hunk 4]
```diff
@@ -70,9 +70,15 @@ public static Map<String, Object> execute(Map<String, String> params) throws Nam
...
+                log.error("exception in main - {}",err.get("exception"));
```
Comment (plyubich) [inline jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java:80]: logging consistency

exception in main - {} -> Exception in Main : {}

**Review by plyubich — CHANGES_REQUESTED** [review]
@raviteja-mandala please be consistent in your logging messages. In this PR each log message was slightly different. All logs should follow the same format.

Lastly, please apply formatting standards.

**Comment by plyubich** [comment]
@raviteja-mandala this PR needs a good description and a screenshots of an error from Qualys in cloudwatch and datadog, and an Alert in slack.

File: jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/util/ErrorManageUtil.java (hunk 2 above)
Comment (raviteja-mandala) [reply jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/util/ErrorManageUtil.java:38]: These logger statements are temporarily included to debug qualys data alert issue. They will be reverted once the issue is resolved. In qualys, we are getting data alert but when we check logs, we are not finding any error/exception which caused this data alert. To know what error caused this, we have included the above logger statements.

File: jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java (hunk 3 above)
Comment (raviteja-mandala) [reply jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java:79]: These logger statements are temporarily included to debug qualys data alert issue. They will be reverted once the issue is resolved. In qualys, we are getting data alert but when we check logs, we are not finding any error/exception which caused this data alert. To know what error caused this, we have included the above logger statements.

File: jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java (hunk 4 above)
Comment (raviteja-mandala) [reply jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java:80]: These logger statements are temporarily included to debug qualys data alert issue. They will be reverted once the issue is resolved. In qualys, we are getting data alert but when we check logs, we are not finding any error/exception which caused this data alert. To know what error caused this, we have included the above logger statements.

File: jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/util/ErrorManageUtil.java (hunk 2 above)
Comment (raviteja-mandala) [reply jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/util/ErrorManageUtil.java]: reformatted.

File: jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java (hunk 3 above)
Comment (raviteja-mandala) [reply jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java]: reformatted.

File: jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java (hunk 4 above)
Comment (raviteja-mandala) [reply jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java]: done.

**Review by santhosh-challa — APPROVED** [review]

**Comment by santhosh-challa** [comment]
@plyubich merging this for now as this contains debug logs only. We will revert this.

**Pull request merged**